### ISSUE TITLE
Feature: Footer links should have an underline

### ISSUE THREAD

**Issue by akshaybabloo** [vuejs/vitepress#3171]
### Is your feature request related to a problem? Please describe.

Currently the links in the footer doesn't show that they are links, i.e. they look like simple texts without underline or hover highlight

### Describe the solution you'd like

It would be nice to have a way to highlight the links when using the [footer](https://vitepress.dev/reference/default-theme-footer) theme configuration. 

### Describe alternatives you've considered

I am adding a CSS field to highlight them for now

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Read the [Contributing Guidelines](https://github.com/vuejs/vitepress/blob/main/.github/contributing.md).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.
